gameplay/src/Quaternion.h

```diff
@@ -263,6 +272,13 @@ class Quaternion
      */
     void set(const Vector3& axis, float angle);
 
+	/**
+	* Sets the elements of this quaternion to a copy of the specified euler angle.
+	*
+	* @param euler The euler angle (in radians).
```

Comment:
Perhaps use the same wording here as you did for createFromEuler. Otherwise it sounds like it's a single angle rather than a series of 3 rotations.